packages/typespec-azure-core/test/rules/no-enum.test.ts

```diff
@@ -46,6 +47,27 @@ describe("typespec-azure-core: no-enum rule", () => {
       .toBeValid();
   });
 
+  it("emit warning about other enums in versioned service", async () => {
```

Comment:
I am wondering why we have this warning in azure scope?

Looking at the swagger in mgmt, there are a lot of cases where `"modelAsString": false`.
https://github.com/Azure/azure-rest-api-specs/blob/feature/health-lro/specification/compute/resource-manager/Microsoft.Compute/CloudserviceRP/preview/2020-10-01-preview/cloudService.json#L1483-L1492

Reply:
Being an azure scope is the entire point of this linting rule. Actual enums cannot deal with new values being added so azure policy is you shouldn’t use them. There is of course always exceptions.

Reply:
I see this guideline: https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#enums--sdks-client-libraries, is it from there?